Makefile

```diff
@@ -49,6 +49,21 @@ setup-config:
 	@read -p "Enter your LLM API key: " llm_api_key; \
 	 echo "LLM_API_KEY=\"$$llm_api_key\"" >> $(CONFIG_FILE).tmp
 
+	@echo "Enter your LLM Embedding Model\nChoices are openai, azureopenai, llama2 or leave blank to default to 'BAAI/bge-small-en-v1.5' via huggingface"; \
+	 read -p "> " llm_embedding_model; \
+	 echo "LLM_EMBEDDING_MODEL=\"$$llm_embedding_model\"" >> $(CONFIG_FILE).tmp; \
+	 if [ "$$llm_embedding_model" = "llama2" ]; then \
+	 read -p "Enter the local model URL: " llm_base_url; \
+	 echo "LLM_BASE_URL=\"$$llm_base_url\"" >> $(CONFIG_FILE).tmp; \
+	 elif [ "$$llm_embedding_model" = "azureopenai" ]; then \
+	 read -p "Enter the Azure endpoint URL: " llm_base_url; \
+	 echo "LLM_BASE_URL=\"$$llm_base_url\"" >> $(CONFIG_FILE).tmp; \
+	 read -p "Enter the Azure LLM Deployment Name: " llm_deployment_name; \
+	 echo "LLM_DEPLOYMENT_NAME=\"$$llm_deployment_name\"" >> $(CONFIG_FILE).tmp; \
+	 read -p "Enter the Azure API Version: " llm_api_version; \
```

Comment:
possible to indent to make this more readable?

Reply:
Yes will work on that and resubmit

Reply:
Indent added. Let me know if that is okay.